### PR TITLE
Finish first version of the scripts

### PR DESCRIPTION
**Pull request by AdamHawtin**
# Motivation and Context
The initial commit copied in a working but unfinished version of the batch runner, this PR is to finish it. For this review please raise any issues with the code on master as well.

This is a tool for generating messages to request unaddressed QID/UAC pairs, it takes a CSV config file specifying the questionnaire types and quantities.

# What has Changed
- Add Dockerfile and dockerignore
- Add travis build to test, build and push image
- Add scripts and instructions for running in k8s
- Fix default configs
- Copy in rabbit context unit tests

# How to Test
- Follow the README instructions for running in K8s, check you can successfully queue all the qid request messages

# Links
https://trello.com/c/zRjY6YpP/714-unaddressed-batch-runner-8

### CODE REVIEW

**Comment by AdamHawtin** [comment]
[Unaddressed Batch Runner (8)](https://trello.com/c/zRjY6YpP/714-unaddressed-batch-runner-8)

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

File: README.md [hunk 1]
```diff
@@ -25,9 +25,26 @@ Run the script locally with
 pipenv run python generate_qid_batch.py <path to config csv>
 ```
 
-<!--->### In Kubernetes TODO<--->
+### In kubernetes
+To start up the image and connect to it's shell in Kubernetes, point your kubectl context at the cluster you wish to run in, then run
```
Comment (jcox-dev) [inline README.md]:
```suggestion
To start up the image and connect to its shell in Kubernetes, point your kubectl context at the cluster you wish to run in, then run
```

File: makefile [hunk 2]
```diff
@@ -9,3 +9,9 @@ lint:
 
 test: check lint
 	pipenv run pytest
+
```
Comment (jcox-dev) [inline makefile]: Can this be renamed to `Makefile`?

**Review by jcox-dev — APPROVED** [review]
Other than a couple of cosmetic changes, this works as expected - locally, and in a GCP k8s cluster.

**Review by liamtoozer — APPROVED** [review]

**Pull request merged**